.provenance/SPEC.md

```
@@ -0,0 +1,90 @@
+
+```bash
+bun run prov start "Fix stale selection after menu close"   # prints the change id
+printf '%s' 'when the menu closes the selected workspace must stay selected' \
+  | bun run prov human <change>                             # exact words, on stdin
+bun run prov decide <change> "Move selection ownership into WorkspaceStore" \
+  --component frontend --detail "why, at whatever length"   # or --detail - to read stdin
+```
+
+Then every commit for that task carries the trailer. Git only parses the last paragraph of a message, so it goes in the same block as any other trailer, with no blank line between them:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)